### PR TITLE
Add metric monitoring doc

### PR DESCRIPTION
**Pull request by simon-mo**
Add metric monitoring under tutorial section. A quick note on how to access it. (as part of 0.3.0 release)

### CODE REVIEW

File: content/tutorials/metrics.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
++++
+title= "Metric Monitoring"
+date= 2018-03-13T10:23:10-07:00
+description = ""
+draft= false
++++
+
+In starting Clipper v0.3.0, Clipper report its metrics to a prometheus server.
```
Comment (dcrankshaw) [inline content/tutorials/metrics.md]: Change this sentence to: "Starting with version 0.3.x, Clipper reports metrics to a Prometheus server"

Add a link to the Prometheus documentation here as well.

File: content/tutorials/metrics.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+In starting Clipper v0.3.0, Clipper report its metrics to a prometheus server. 
+
+## Prometheus Server
+We use prometheus as the metric tracking system. Once you spin up a clipper query frontend and a model containers.
+If you are using `DockerContainerManager`, You can view prometheus UI at: [`http://localhost:9090`](http://localhost:9090).
+If you are using `KubernetesContainerManager`, You can query the metric address by calling `get_metric_addr()`.
+
+Please note that Prometheus UI is for debug purpose only. You can view certain metric and graph the timeseries. But for better visualization, we recommend [Grafana](https://grafana.com/). Grafana has default support for Prometheus Client. Feel free to checkout `examples/monitoring` for example of displaying Clipper metrics in Grafana.
+
+## Avaliable Metrics
```
Comment (dcrankshaw) [inline content/tutorials/metrics.md]: Available is spelled wrong

**Comment by simon-mo** [comment]
@dcrankshaw fixed!

**Comment by dcrankshaw** [comment]
You're still missing the link to Prometheus website

**Pull request merged**